### PR TITLE
Fix select field to allow programmatically setting value to undefined

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Addresses #43

### CODE REVIEW

File: src/app/visual/select-field/select-field-visual.component.html [hunk 1]
```diff
@@ -51,11 +63,39 @@ <h2>Multiple select mode</h2>
...
+  </table>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="model.multiple = undefined"
+  >
+    Set ngModel to undefined
+  </button>
+
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/select-field/select-field-visual.component.html:119]: ☝️ 2 changes to the visual test:
1. Changed `ngModel` to be 2-way bound so setting it to `undefined` would register as a change.
2. Added the nicety of an output table so devs can see how the `ngModel` properties are updated.

File: src/app/public/modules/select-field/select-field.component.ts [hunk 2]
```diff
@@ -210,11 +210,9 @@ export class SkySelectFieldComponent implements ControlValueAccessor, OnDestroy
       return;
     }
 
-    if (value) {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/select-field/select-field.component.ts:213]: We weren't letting anything `undefined` through the CVA. Removing this `if` check allows consumers to programmatically change the value to `undefined`. However, we had to add one check for tokens in this case (see below).

File: package.json [hunk 3]
```diff
@@ -36,53 +36,53 @@
...
+    "@angular/http": "7.2.16",
+    "@angular/platform-browser": "7.2.16",
+    "@angular/platform-browser-dynamic": "7.2.16",
+    "@angular/router": "7.2.16",
+    "@blackbaud/auth-client": "2.22.0",
+    "@pact-foundation/pact": "9.6.1",
     "@pact-foundation/pact-web": "7.4.0",
-    "@skyux-sdk/builder": "3.10.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0",
+    "@skyux-sdk/builder": "3.14.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: Mind bumping this to `3.15.0`. It includes an important feature for libraries.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Just a minor pipeline suggestion; overall looks great. Thanks for the detailed PR comments!

File: package.json (hunk 3 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: Done.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**